packages/framework-provider-azure/src/library/events-stream-consumer-adapter.ts

```diff
@@ -0,0 +1,65 @@
+import { BoosterConfig, EventEnvelope, EventStream } from '@boostercloud/framework-types'
+import { Context } from '@azure/functions'
+import { getLogger } from '@boostercloud/framework-common-helpers'
+import { AZURE_CONFLICT_ERROR_CODE } from '../constants'
+import { CosmosClient } from '@azure/cosmos'
+
+interface DedupEventStream {
+  primaryKey: string
+  createdAt: string
+  ttl: number
+}
+
+export async function dedupEventStream(
+  cosmosDb: CosmosClient,
+  config: BoosterConfig,
+  context: Context
+): Promise<EventStream> {
+  const logger = getLogger(config, 'dedup-events-stream#dedupEventsStream')
+  const events = (context.bindings.eventHubMessages as EventStream) || []
+  logger.debug(`Dedup ${events.length} events`)
+
+  const resources: EventStream = []
+  for (const event of events) {
+    const rawParsed = JSON.parse(event as string)
+    const eventTag: DedupEventStream = {
+      primaryKey: rawParsed._etag,
+      createdAt: new Date().toISOString(),
+      ttl: config.eventStreamConfiguration.parameters?.dedupTtl ?? 86400,
```

Comment:
Two things:
- Great point on being able to define the TTL
- A bit nitpicking: I guess this default TTL is in seconds (24 hours). Could you please put this in a constant with a name?